From 8c11f207e2bbf0f98488de7066bd3698705f20e1 Mon Sep 17 00:00:00 2001
From: Ben Cooksley <bcooksley@kde.org>
Date: Mon, 7 Feb 2022 06:44:47 +1300
Subject: [PATCH] Use the correct endpoint for this - hosted by a CDN and
 therefore capable of handling much greater volumes of traffic.

CCMAIL: plasma-devel@kde.org
CCMAIL: distributions@kde.org
---
 kcms/kfontinst/kcmfontinst/kfontinst.knsrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kcms/kfontinst/kcmfontinst/kfontinst.knsrc b/kcms/kfontinst/kcmfontinst/kfontinst.knsrc
index deaf1e08f..932bc8802 100644
--- a/kcms/kfontinst/kcmfontinst/kfontinst.knsrc
+++ b/kcms/kfontinst/kcmfontinst/kfontinst.knsrc
@@ -41,7 +41,7 @@ Name[vi]=Phông chữ
 Name[x-test]=xxFontsxx
 Name[zh_CN]=字体
 
-ProvidersUrl=https://distribute.kde.org/khotnewstuff/fonts-providers.xml
+ProvidersUrl=https://cdn.kde.org/khotnewstuff/v1/fonts-providers.xml
 Categories=KDE-Look.org Fonts
 TargetDir=kfontinst
 Uncompress=archive
-- 
GitLab

